addons/test_mail/tests/test_performance.py

```diff
@@ -67,6 +67,8 @@ def setUpClass(cls):
 
     def setUp(self):
         super(BaseMailPerformance, self).setUp()
+        # enable email sending for query counters compatibility after odoo/odoo#186884
+        self.patch(type(self.env["ir.mail_server"]), "_disable_send", lambda _: False)
```

Comment:
You have enabled sending, but there is no mock server. I suppose that is why it crashes.